tests/column_resizer_tester.ts

```diff
@@ -0,0 +1,264 @@
+import { Expect, Test } from 'alsatian';
+import { ColumnResizer, TableInterface } from '../source';
+
+class FauxTableInterface implements TableInterface {
+  constructor() {
+    this._columnCount = 3;
+    this._activeWidth = 20;
+    this._widths = [200, 300, 200];
+    this._corners = {topLeft: {x: 0,y: 0}, bottomRight: {x: 700,y: 100}};
+  }
+
+  public get columnCount() {
+    return this._columnCount;
+  }
+
+  public get activeWidth() {
+    return this._activeWidth;
+   }
+
+  public get corners() {
+    return this._corners;
+  }
+
+  public getColumnWidth(index: number) {
+    return this._widths[index];
+  }
+
+  public onResize (columnIndex: number, difference: number){
+    if(columnIndex >= this._columnCount) {
+      throw RangeError;
+    }
+    if(difference === 0){
```

Comment:
Space before ```{```.

Always run a linter before submitting a code review. These should all be caught by a linter.